jwt/scalewayJWT.js

```diff
@@ -0,0 +1,49 @@
+const BbPromise = require('bluebird');
+const setUpDeployment = require('../shared/setUpDeployment');
+const getJwt = require('./lib/getJwt');
+const { Api } = require('../shared/api');
+
+class ScalewayJwt {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options || {};
+    this.provider = this.serverless.getProvider('scaleway');
+    this.provider.initialize(this.serverless, this.options);
+
+    this.commands = {
+      jwt: {
+        usage: 'Get JWT Token',
+        lifecycleEvents: ['jwt'],
+        commands: {
+          start: {
+            usage: 'Get JWT tokens for your namespace and your private functions/containers.',
+            lifecycleEvents: [
+              'jwt',
+            ],
+          },
+        },
+      },
+    };
+
+    const credentials = this.provider.getCredentials();
+    const api = new Api(credentials.apiUrl, credentials.token);
+
+    Object.assign(
+      this,
+      setUpDeployment,
+      getJwt,
+      api,
+    );
+
+    this.hooks = {
```

Comment:
this.commands and this.hooks should be next to each other, for code organization (informations about the command and its execution should be grouped)